test/select.spec.js

```diff
@@ -2093,6 +2095,20 @@ describe('ui-select tests', function() {
 
       expect($(el).scope().$select.selected.length).toBe(5);
     });
+
+    it('should add an id to the search input field', function () {
+      var el = createUiSelectMultiple({inputId: 'inid'});
+      var searchEl = $(el).find('input.ui-select-search');
+      expect(searchEl.length).toEqual(1);
+      expect(searchEl[0].id).toEqual('inid');
+    });
+  });
+
+  it('should add an id to the search input field', function () {
```

Comment:
I wanted to test both single and multiple because they are very different beasts (even though they both have the same text input name right now). The `it` name is the same because the `describe` names will disambiguate them. I'm happy to remove one if you want.
I'd prefer to keep the multiple test because that's the one I care about most (and that search field is always on display so much more likely to be used for `<label for="...">`.